src/views/Home/components/Mission/mission.scss

```diff
@@ -1,3 +1,108 @@
-/* stylelint-disable-line no-empty-source */
+@use "~stylesheets/responsiveness";
+@use "~stylesheets/typography";
+@use "~stylesheets/variables";
 
-/* Delete these lines when you begin development */
+.Mission {
+  position: relative;
+
+  /* Text Layer */
+  &--text {
+    display: flex;
+    flex-direction: column;
+    background: url("~images/mission-background.png");
```

Comment:
check out the changes made [here](https://github.com/CruzHacks/cruzhacks-2021-website/pull/91#discussion_r536441553) to remove the hardcoded background color & add transparency while keeping the grid svg— this will let the page-level background float visibly underneath
![Screenshot from 2020-12-19 13-44-22](https://user-images.githubusercontent.com/19658882/102700224-682aca00-4200-11eb-9bf6-ab4ae82234f8.png)

Reply:
Thanks for letting me know about that - it looks so much better!